configuration/persistence.md

```diff
@@ -11,59 +11,73 @@ openHAB can store data over time; this is known as persistence.
+You may have more than one persistence add-on loaded, and each of these may be configured independently.
 
 A complete list of supported persistence add-ons may be found in the [persistence]({{base}}/addons/persistence.html) section of the on-line openHAB documentation.
 
-## Persistence Add-on Configuration
+## Add-on Configuration
 
-Each persistence add-on you install will need to be configured.
 Please refer to the specific [on-line documentation]({{base}}/addons/persistence.html) for your selected persistence add-on for configuration instructions.
 
 ## Default Persistence Service
 
-You may install more than one persistence add-on.
-Therefore, it is important to select a default persistence service.
+It is important to select a default persistence service.
 You should do this even if you have only one persistence add-on installed.
 
 To select a default persistence service, in paper UI, select Configuration and then System from the side menu.
-Scroll down to "Persistence", and select your Default Service from the drop-down list.
+Scroll down to "Persistence", and select your default service from the drop-down list.
 Note that you must first install a persistence add-on before you make this selection.
 Be sure to save your choice once you have selected your default service.
 
-## Configuration
+## Persistence Configuration
 
-Persistence Strategies are configured in a file named `<persistenceservice>.persist`, stored in `$OPENHAB_CONF/persistence` ("persistenceservice" is replaced by the name of your add-on (e.g. `rrd4j.persist`)).
+Persistence Strategies are configured in a file named `<persistenceservice>.persist`, stored in `$OPENHAB_CONF/persistence` (Replace "persistenceservice" with the name of your persistence add-on (e.g. `rrd4j.persist`)).
 
-Strategies may be used to persist an Item state when some event has occurred (e.g. an Item state has been updated or changed), on a time schedule, or at a specific time of day (e.g. through a [cron expression](http://www.quartz-scheduler.org/documentation/quartz-2.1.x/tutorials/crontrigger).
+### Persistence Triggers
 
-Persistence configuration files consist of several sections:
+The persistence of an Item's state may be triggered when that Item changes state or when the Item is updated (even if its state did not change).
+Persistence may also be triggered by a time-related event (see Cron Persistence Triggers below).
 
-### Strategies section
+### Strategies
 
-This section allows you to define strategies and to declare a set of default strategies to use for this persistence service.  The syntax is as follows:
+This section allows you to define one or more `Strategies` that will be applied to `Items` (you specifiy the `Items` in the `Items` section).
+This section also defines a default strategy.
+The syntax is as follows:
 
 ```java
 Strategies {
   <strategyName1> : "cronexpression1>"
+  <strategyName2> : "cronexpression2"
```

Comment:
With our without `<>`?

Reply:
Right - ERROR.  Will change to <strategyName1> : "cronexpression1"